src/bosh_azure_cpi/lib/cloud/azure/cloud.rb

```diff
@@ -743,9 +743,10 @@ def _init_azure
       cloud_error("Please make sure the CPI has proper network access to Azure. #{e.inspect}") # TODO: Will it throw the error when initializing the client and manager
     end
 
-    def _init_cpi_lock_dir
-      CPILogger.instance.logger.info('_init_cpi_lock_dir: Initializing the CPI lock directory')
+    def _init_cpi_dirs
+      CPILogger.instance.logger.info('_init_cpi_dirs: Initializing the CPI lock directory')
```

Comment:
`Initializing the CPI lock directory` => `Initializing the CPI directory`

Reply:
changed.